package_manifest.yaml

```diff
@@ -776,6 +776,8 @@ foreman_client_packages:
   hosts:
     gofer: {}
     katello-host-tools: {}
+    katello-pull-provider-migrate: {}
+    python-beautifulsoup4: {}
```

Comment:
this seems not needed in this PR? (it was a tracer dep, but we dropped it, iirc)

Reply:
Missed that one.  removed.